src/auth.js

```diff
@@ -0,0 +1,21 @@
+import { get, readable } from "svelte/store";
+var value = "LOADING";
+var obj;
+export const state = readable(value, function auth(set) {
+  fetch("https://auth.canadiana.ca/1/auth", {
+    credentials: "include"
+  })
+    .then(function(response) {
+      if (response.status >= 200 && response.status <= 300) {
+        value = "SUCCESS";
+        set(value);
+      } else if (response.status >= 400 && response.status <= 500) {
```

Comment:
Anything above 400 is a failure.

```suggestion
      } else if (response.status >= 400) {
```